pxc/jenkins/qa_pxc_cross-pipeline.groovy

```diff
@@ -0,0 +1,175 @@
+pipeline {
```

Comment:
I would also add the scm checkout part in groovy before pipeline part. For example as in https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pxb/jenkins/pxb-package-testing.groovy:
`library changelog: false, identifier: 'lib@master', retriever: modernSCM([
    $class: 'GitSCMSource',
    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
]) _`

Reply:
Fixed !

Reply:
Fixed